packages/firestore/src/api/database.ts

```diff
@@ -415,6 +432,16 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
     return this._firestoreClient as FirestoreClient;
   }
 
+  private getDatabaseInfo(): DatabaseInfo {
```

Comment:
Since this isn't a getter exactly, consider naming this with a `make` or `create` prefix. Try to be consistent with other examples nearby (e.g. `makeCredentialsProvider`).